index.html

```diff
@@ -0,0 +1,447 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <nav>
+      <div id="nav-bar">
+        <header class="nav-cont container">
+          <a id="logo" href="#">Aziz Mejri</a>
+          <button type="submit" class="menu-bars">
+            <i class="fas fa-bars"></i>
+          </button>
+          <ul id="nav-list">
+            <div class="close-toggle" id="close-toggle">
+              <i class="fa fa-times" aria-hidden="true"></i>
+            </div>
+            <li><a href="#projects" class="nav-link">Portfolio</a></li>
+            <li><a href="#about" class="nav-link">About</a></li>
+            <li>
+              <a href="#contact" class="nav-link"
+                >Contact <i class="far fa-envelope"></i
+              ></a>
+            </li>
+          </ul>
+        </header>
+      </div>
+    </nav>
+    <main>
```

Comment:
As requested by the previous reviewer, please wrap all your header elements inside the `header`. This means that your logo and the `nav` should be inside the `header`. This will help in accessibility and styling your links 😄.